share/picongpu/tests/Shadowgraphy/include/picongpu/param/incidentField.param

```diff
@@ -37,11 +37,11 @@ namespace picongpu
             {
                 struct MyGaussianParam
                 {
-                    static constexpr float_64 WAVE_LENGTH_SI = 800e-9; // TIME_PERIOD_SI * SI::SPEED_OF_LIGHT_SI;
-                    static constexpr float_64 TIME_PERIOD_SI = WAVE_LENGTH_SI / SI::SPEED_OF_LIGHT_SI;
+                    static constexpr float_64 WAVE_LENGTH_SI = 800e-9; // TIME_PERIOD_SI * sim.si.getSpeedOfLight();
```

Comment:
with auto replacement I can not take care, thats why I always say we should not have dead code in files. But maybe it is in a comment because in the past it was at this point in the param files not possible to access DELTA_T and SPEED_OF_LIGHT.